lib/tasks/redis.rake

```diff
@@ -0,0 +1,22 @@
+require "redis"
+
+namespace :redis do
+  desc "Make redis a slave of production redis"
+  task :make_slave => :environment do
+    redis = Redis.new(url: ENV.fetch('REDIS_URL'), :driver => :hiredis)
+    redis.slaveof(ENV.fetch('PRODUCTION_REDIS_HOST'), ENV.fetch('PRODUCTION_REDIS_PORT'))
```

Comment:
I think you can parse the `REDIS_URL` variable to get the constituent parts to keep the env vars DRY.

```
prod_redis_uri = URI.parse(ENV["PRODUCTION_REDIS_URL"])

redis.slaveof(ENV.fetch(uri.host), ENV.fetch(uri.port))
redis.config('set', 'masterauth', ENV.fetch(uri.password))
```